src/runner/mock-test-runner.js

```diff
@@ -129,3 +128,20 @@ export class MockTestRunner {
     };
   }
 }
+
+/**
+ * Summarize cli options as mock options for creating a test runner.
+ * @param {object} options
+ * @param {boolean} options.mock
+ * @param {'request' | 'skip'} options.mockOpenPage
+ * @returns {AriaATCIRunner.MockOptions}
+ */
+export function runnerMockOptions({ mock, mockOpenPage }) {
+  if (mock === undefined && mockOpenPage) {
```

Comment:
In order to support this branch, the type for `mock` should read something like `@param {boolean} [options.mock]`, and the same goes for the optional `mockOpenPage` parameter. That said, I'm wondering if we need to implement these branches in the first place. I personally tend to prefer explicitness over brevity, and in this case, the shorthand doesn't seem to be in use--the types used in the call site indicate that these values are always specified.

Reply:
Take a look at [75b5aa1](https://github.com/w3c/aria-at-automation-harness/pull/62/commits/75b5aa1218c59bcd51a49880d890761d5a47474e). We weren't using the mock "config" in any meaningful way, and on closer look it struck as a classic case of YAGNI. I think we can get away with just a boolean CLI arg to determine whether or not to run the mock server.